<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathPlanning</class>
 <widget class="QMainWindow" name="PathPlanning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PathPlanning</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frameViewer">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Scenes</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButtonScene1">
             <property name="text">
              <string>Load Scene 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonScene2">
             <property name="text">
              <string>Load Scene 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonScene3">
             <property name="text">
              <string>Load Scene 3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonScene4">
             <property name="text">
              <string>Load Scene 4</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonReset">
             <property name="text">
              <string>Reset Scene</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Exercises</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButtonExercise1">
             <property name="text">
              <string>Exercise 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonExercise2">
             <property name="text">
              <string>Exercise 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonExercise3">
             <property name="text">
              <string>Exercise 3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonExercise4">
             <property name="text">
              <string>Exercise 4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonExercise5">
             <property name="text">
              <string>Exercise 5</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Visualization</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="checkBoxColModel">
             <property name="text">
              <string>Robot Collision Model</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxVisualizeIK">
             <property name="text">
              <string>Visualize IK</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxTCPCoordSystems">
             <property name="text">
              <string>TCP Coordinate Systems</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1103</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
